flink-core/src/test/java/org/apache/flink/api/java/typeutils/runtime/kryo/KryoSerializerUpgradeTest.java

```diff
@@ -31,44 +31,53 @@
 
 import com.esotericsoftware.kryo.serializers.DefaultSerializers;
 import org.assertj.core.api.Condition;
+import org.junit.jupiter.api.Disabled;
 
 import java.util.ArrayList;
 import java.util.Collection;
 
 import static org.apache.flink.api.common.typeutils.TypeSerializerConditions.hasSameCompatibilityAs;
 import static org.apache.flink.api.common.typeutils.TypeSerializerSchemaCompatibility.compatibleWithReconfiguredSerializer;
 
+// These tests are disabled because Flink 2.0 is doing a major Kryo 2.x -> 5.x upgrade and
+// the new Flink is not compatible with older Kryo state.
+// Restore tests when there is a Flink 2.1 that should test compatibility with Flink 2.0
```

Comment:
Can we open a blocker ticket for 2.1 on JIRA to track this? Otherwise it may get easily forgotten

Reply:
Is this a good blocker ticket for both of the tests you flagged?
https://issues.apache.org/jira/browse/FLINK-37404

Reply:
yes, thank you!